### DISCUSSION TITLE
getOrders API function shows incorrect parameter names

### DISCUSSION THREAD

**Discussion by rogerjbos** [ProjectOpenSea/opensea-js#1200] [General]
When I hover over the getOrders function it shows parameters names such as `makerAddress` and `takerAddress`, but those don't work.  They don't provide any filtering.  I saw come code online that shows just `maker` so I tried that and that works perfectly.  Not sure why VS Code is showing the incorrect parameter names.

**Comment by rogerjbos** [comment]
Also, I have two related question about contract address filtering.  For example, this code works in that it returns order only for the specified NFT and token id:
```
let options = {
    assetContractAddress: "0x4135063DC85190660ed08790f59Bc711d8b404c0",
    tokenId: 3333,
    limit: 20,
    side: "bid"
}
const orders = await openseaSDK.api.getOrders(options);
```
What if I wanted to know the order for the collection without specifying the token ids.  If I leave it blank, null, or remove it I get an error.

I am also not able to filter by more than one tokenId using an array as suggested by the docs.  The following code also returns an error:
```
let token_ids = ["1","3333"];
let options = {
    assetContractAddress: "0x4135063DC85190660ed08790f59Bc711d8b404c0",
    tokenId: token_ids,
    limit: 20,
    side: "bid"
}
const orders = await openseaSDK.api.getOrders(options);
```

Could someone please tell me what I am doing wrong?

**Reply by ryanio** [reply]
> What if I wanted to know the order for the collection without specifying the token ids.

You'll want to use https://docs.opensea.io/reference/get_all_listings_on_collection_v2 or https://docs.opensea.io/reference/get_all_offers_on_collection_v2 - I don't believe these are added to opensea-js yet so you can use the general purpose `get` method under the sdk. 

> I am also not able to filter by more than one tokenId using an array as suggested by the docs. The following code also returns an error:

what was the error?

**Comment by ryanio** [comment]
thanks for the report, have opened a PR to fix the param names: https://github.com/ProjectOpenSea/opensea-js/pull/1223
